src/components/totals/TotalDonations/index.js

```diff
@@ -67,8 +71,16 @@ module.exports = React.createClass({
   loadTotals: function() {
     this.setState({ isLoading: true });
 
+    var props        = this.props;
     var campaignUids = this.props.campaignUid || this.props.campaignUids;
-    totals.findByCampaigns(campaignUids, this.onSuccess);
+    var charityUids  = props.charityUids;
```

Comment:
I notice this component can now accept both `charityUid` and `charityUids` but here you only handle the plural version? Should this be like:

```js
 var charityUids  = props.charityUid || props.charityUids;
```